app/blueprints/fund_builder/routes.py

```diff
@@ -630,6 +679,8 @@ def create_export_zip(directory_to_zip, zip_file_name, random_post_fix) -> str:
 
 
 @build_fund_bp.route("/create_export_files/<round_id>", methods=["GET"])
+@login_required(return_app=SupportedApp.FUND_APPLICATION_BUILDER)
+@check_internal_user
```

Comment:
Is this check on essentially every view?

Should we add this in a more centralised place instead, eg a `before_request` check? 🤔  Feels like it'd be very easy to miss a route this way.

Reply:
Fair comment, I did try to centralise initially but got stuck and... opted for the easy option 😶‍🌫️ 

Have now implemented this though